lib/fs.js

```diff
@@ -287,6 +286,7 @@ function existsSync(path) {
   try {
     path = getValidatedPath(path);
   } catch {
+    deprecate(() => {}, 'Passing invalid argument types to fs.existsSync is deprecated', 'DEP0187');
```

Comment:
Can you add a test that validates a warning is emitted once please?
```suggestion
    if (!deprecationWarningEmitted) {
      process.emitWarning('Passing invalid argument types to fs.existsSync is deprecated', 'DeprecationWarning', 'DEP0187');
      deprecationWarningEmitted = true;
    }
```

Reply:
Added!